package.json

```diff
@@ -28,9 +18,9 @@
     "postcommit": "git update-index --again",
     "prerelease": "yarn ci && cross-env NODE_ENV=production yarn build",
     "postrelease": "yarn deploy:docs",
-    "release:major": "yarn prerelease && ta-script npm/release major && yarn postrelease",
-    "release:minor": "yarn prerelease && ta-script npm/release minor && yarn postrelease",
-    "release:patch": "yarn prerelease && ta-script npm/release patch && yarn postrelease",
+    "release:major": "yarn prerelease && lerna publish major && yarn postrelease",
```

Comment:
Remove `ta-scripts` dependency?

Reply:
`ta-script npm/release` does several things - sanity checks, pushes git tag... are you sure `lerna publish` does all these?